### PR TITLE
Add fastlane builder

### PR DESCRIPTION
**Pull request by dlackty**
As titled, please let me know if there's any concern. Thank you!

### CODE REVIEW

**Comment by dlackty** [comment]
Since there have been some significant changes made to `android-builder`, I'll rebase this branch and make an updated example soon.

File: fastlane/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.5.3-alpine
+
+RUN apk add --update --no-cache ruby-dev build-base libxml2-dev libxslt-dev
+RUN gem i fastlane --no-ri --no-rdoc
```
Comment (nof20) [inline fastlane/Dockerfile]: I don't fully understand why, but you need to remove `--no-ri` and `--no-rdoc` to make this work.

**Comment by dlackty** [comment]
@nof20 I've added a commit to resolve the issue, and also rebased the branch onto master.

**Comment by nof20** [comment]
Looks good.  Thanks again for the contribution.

**Pull request merged**